src/test/java/org/mule/test/oauth2/internal/authorizationcode/functional/AuthorizationCodeRefreshTokenWhenConfigTestCase.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.oauth2.internal.authorizationcode.functional;
+
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.mule.runtime.oauth.api.state.ResourceOwnerOAuthContext.DEFAULT_RESOURCE_OWNER_ID;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.mule.extension.http.api.request.validator.ResponseValidatorTypedException;
+import org.mule.extension.oauth2.api.tokenmanager.TokenManagerConfig;
+import org.mule.functional.api.exception.ExpectedError;
+
+public class AuthorizationCodeRefreshTokenWhenConfigTestCase extends AbstractAuthorizationCodeRefreshTokenWhenConfigTestCase {
+
+  @Rule
+  public ExpectedError expectedError = ExpectedError.none();
+
+  public static final String SINGLE_TENANT_OAUTH_CONFIG = "oauthConfig";
+
+  @Override
+  protected String getConfigFile() {
+    return "authorization-code/authorization-code-refresh-token-when-config.xml";
+  }
+
+  @Test
+  public void afterFailureDoRefreshWhenTokenWithDefaultValueNoResourceOwnerId() throws Exception {
+    executeRefreshTokenWhen("testFlowRefreshTokenWhen", SINGLE_TENANT_OAUTH_CONFIG, DEFAULT_RESOURCE_OWNER_ID, 403);
+  }
+
+  @Override
+  protected TokenManagerConfig getTokenManagerConfig() {
+    return registry.<TokenManagerConfig>lookupByName(SINGLE_TENANT_OAUTH_CONFIG).get();
```

Comment:
Check if this can be done with an Inject instead

Reply:
Seems to work. Making the change.